app/stac_api/management/commands/list_asset_uploads.py

```diff
@@ -96,6 +96,7 @@ def are_uploads_equal(s3_upload, db_upload):
             only_s3_uploads = s3_uploads
 
         print(
+            # pylint: disable=possibly-used-before-assignment
```

Comment:
Dito this is not safe to disable this pylint might be a better idea to fix the code